xbmc/video/dialogs/GUIDialogVideoSettings.cpp

```diff
@@ -169,7 +169,7 @@ void CGUIDialogVideoSettings::Save()
     return;
 
   // prompt user if they are sure
-  if (CGUIDialogYesNo::ShowAndGetInput(12376, 750, 0, 12377))
+  if (CGUIDialogYesNo::ShowAndGetInput(12376, 12377, -1, -1))
```

Comment:
Won't work:
	3	IntelliSense: no instance of overloaded function "CGUIDialogYesNo::ShowAndGetInput" matches the argument list
            argument types are: (int, int)	c:\xbmc_code\xbmc\addons\GUIDialogAddonInfo.cpp	238	8	Kodi